src/ExceptionAll.Abstractions/Details/ConflictDetails.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ExceptionAll.Abstractions.Details;
+
+public class ConflictDetails : IExceptionAllDetails
+{
+    public string Title => GetDetails().Title;
```

Comment:
All of these Details classes have a similar body - I think you can make a partial class and have GetDetails be virtual then have these classes inherit that then all you have to do is override GetDetails.

Reply:
How do you envision the partial class? I can see it as a base class like the following: 
![image](https://user-images.githubusercontent.com/38481538/154744636-4193d6ec-40ae-4092-8d29-4e4bd4188c6a.png)

Regardless of implementation, I agree this is a simple and effective change.